scripts/syntax_gen.py

```diff
@@ -1255,5 +1257,250 @@ def generatePyBindings(builddir, alltypes):
         outf.write("}\n")
 
 
+def generateCSTJson(builddir, alltypes):
+    # Make sure the output directory exists.
+    headerdir = os.path.join(builddir, "slang", "syntax")
+    try:
+        os.makedirs(headerdir)
+    except OSError:
+        pass
+
+    # Start the header file.
+    outf = open(os.path.join(headerdir, "CSTJsonConverter.h"), "w")
+    outf.write(
+        """//------------------------------------------------------------------------------
+//! @file CSTJsonConverter.h
```

Comment:
I would prefer that this be called CSTSerializer to match ASTSerializer. Also it looks like this class doesn't need to be generated. You can just put it in the library proper and only generate the CSTJsonVisitor into its own header that gets included by CSTSerializer.cpp.